apps/extension/src/pages/activities/index.tsx

```diff
@@ -115,16 +115,24 @@ export const ActivitiesPage: FunctionComponent = observer(() => {
   otherBech32Addresses.setSupportedChainList(supportedChainList);
 
   const msgHistory = usePaginatedCursorQuery<ResMsgsHistory>(
-    process.env["KEPLR_EXT_TX_HISTORY_BASE_URL"],
+    process.env["KEPLR_EXT_TX_HISTORY_TEST_BASE_URL"],
     () => {
       return `/history/msgs/keplr-multi-chain?baseBech32Address=${
         account.bech32Address
       }&chainIdentifiers=${(() => {
         if (selectedKey === "__all__") {
           return supportedChainList
-            .map((chainInfo) => chainInfo.chainId)
+            .map((chainInfo) => {
+              if (chainInfo.chainId === "noble-1") {
+                return "grand-1";
+              }
```

Comment:
이거 임시로 이렇게 하신거면 마킹 해놓는게 좋겠네요
```suggestion
              // TODO: 릴리즈 하기 전에 메인넷으로 돌려놔야함
              if (chainInfo.chainId === "noble-1") {
                return "grand-1";
              }
```

Reply:
여기 외에도 테스트용으로 변경해둔 곳이 많아서 8271422b89ce29e0b8607e5036bd351b725cc453 릴리즈 전 이 커밋 전체를 리무브해야 합니다!

Reply:
네넵!